src/components/Leaderboard/index.tsx

```diff
@@ -1,27 +1,12 @@
-import SegmentDisplay from "./components/Segments";
+import {LeaderboardData} from "@interfaces/Leaderboard";
 
-export default function Leaderboard() {
-  const lap8 = [
-    {
-      meeting_key: 1250,
-      session_key: 9644,
-      driver_number: 63,
-      i1_speed: 182,
-      i2_speed: 208,
-      st_speed: 319,
-      date_start: "2024-11-24T06:43:24.463000+00:00",
-      lap_duration: 97.724,
-      is_pit_out_lap: false,
-      duration_sector_1: 27.928,
-      duration_sector_2: 32.843,
-      duration_sector_3: 36.953,
-      segments_sector_1: [null, 2048, 2048, 2048, 2048, 2048],
-      segments_sector_2: [2048, 2048, 2048, 2048, 2048, 2048, 2051],
-      segments_sector_3: [2048, 2048, 2048, 2048, 2048, 2048, 2049, 2049],
-      lap_number: 25,
-    },
-  ];
+import SegmentDisplay from "./components/Segments";
 
+export default function Leaderboard({
+  leaderboard,
+}: {
+  leaderboard: LeaderboardData;
+}) {
```

Comment:
Mas consulta que otra cosa, es necesaria esa destructuracion?

```suggestion
export default function Leaderboard( leaderboard: LeaderboardData ) {
```

Reply:
Sip, porque al ser un componente no le podemos pasar otra cosa que no sea un objeto, por ahí a futuro si le añadimos más cosas (o no) podríamos pasarlo a una interfaz aparte

Reply:
nt